application/src/main/java/com/sanctionco/thunder/resources/VerificationResource.java

```diff
@@ -331,6 +413,26 @@ public Response resetVerificationStatus(@Auth Key key,
   @GET
   @Path("/success")
   @Produces(MediaType.TEXT_HTML)
+  @Operation(
+      summary = "Get success HTML",
+      description = "Returns an HTML success page that is shown after a user successfully "
+          + "verifies their account. GET /verify will redirect to this URL if the response_type "
+          + "query parameter is set to html.",
+      tags = { "verify" },
+      responses = {
+          @ApiResponse(responseCode = "200",
+              description = "The operation was successful",
+              content = @Content(
+                  mediaType = "text/html", schema = @Schema(example = "<!DOCTYPE html>\n"
+                  + "<html>\n"
+                  + "  <div class=\"alert alert-success\">\n"
+                  + "    <div align=\"center\"><strong>Success!</strong><br>Your account has been "
+                  + "verified.</div>\n"
+                  + "  </div>\n"
+                  + "  <link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/"
+                  + "3.3.7/css/bootstrap.min.css\" />\n"
```

Comment:
This version number seems fairly dynamic. Is there anyway to have this updated over time? This will be especially true if bootstrap has javascript vulnerabilities over time.

Reply:
Yeah that's a good point. I'll have to think about how to do that. I'll open an issue to track it, but in the meantime we'll just have to manually update this version.

#393 is the created issue

Reply:
Just manually upgraded bootstrap to the latest version (`v4.3.1`) - the success page actually looks nicer now so thanks for this catch! We'll figure out how to automate these updates at some point.

Reply:
@NickEckert let me know if this is good to go!